### ISSUE TITLE
[2.1] Add functionality to kingfisher-cli to use the ocds-merge release->compiledRelease functionality to a named collection

### ISSUE THREAD

**Issue by robredpath** [open-contracting/kingfisher#264]
- Refactor OCDSKit to expose a library function to do this (#268)
- Make kingfisher-cli use this (maybe by default)

**Comment by jpmckinney** [comment]
This depends on the answer to #269. In my proposal, one pattern would be:

1. download data to disk
2. transform it however you need with `ocdskit`
3. load into database with `ocdskingfisher`

Another pattern might be either having a Flask app receive data from the scraper (and then do something similar to steps 2-3), OR using `fswatch` on the files/directories written by the scraper, and running steps 2-3 as above.

In other words, it's not necessarily the case that we need a command on `ocdskingfisher` for compiling releases.

**Comment by jpmckinney** [comment]
#272 is related.

**Comment by jpmckinney** [comment]
Note that OCDS Kit now automatically determines which release schema to use (1.0 vs 1.1).

I'm not sure if Kingfisher's merging implementation will take packages or releases as input. If it takes packages, then features like the one above can/should be made reusable within OCDS Kit. If it works directly with releases, then it will have to reproduce the features. So far the only additional feature is choosing the correct version of the release schema, but there's an open issue to merge extensions into the release schema: https://github.com/open-contracting/ocdskit/issues/85

**Comment by odscjames** [comment]
This has been done in https://travis-ci.org/open-contracting/kingfisher-process as a transform, using OCDSMerge directly. Follow up issues to improve on things are in that repository for future work, so closing this issue.
